lib/clamp-device.in

```diff
@@ -176,13 +176,6 @@ if [ $KMDUMPLLVM == "1" ]; then
 fi
 
 # Invoke HCC-specific opt passes
-if [ "$HCC_TILECHECK" == "ON" ]; then
-  $OPT -load $LIB/LLVMSelectAcceleratorCode@CMAKE_SHARED_LIBRARY_SUFFIX@ \
-    -load $LIB/LLVMPromotePointerKernArgsToGlobal@CMAKE_SHARED_LIBRARY_SUFFIX@ \
-    -select-accelerator-code -promote-pointer-kernargs-to-global \
-    -dce -globaldce -always-inline -infer-address-spaces -tile_uniform \
-    < $2.linked.bc -o $2.selected.bc
-else
```

Comment:
This broke the build because the`fi` matching the `else` statement also needs to be removed.
Also please fix the indentation

Reply:
removed the "fi"

Reply:
fixed the indenting.